### PR TITLE
PSR-213 Added individual Nino page

### PR DESCRIPTION
**Pull request by westwater**

### CODE REVIEW

File: app/forms/TextFormProvider.scala [hunk 1]
```diff
@@ -17,22 +17,22 @@
...
 
 import javax.inject.Inject
 
-class TextFormProvider @Inject()() extends Mappings {
+class TextFormProvider @Inject()() {
 
-  val nameRegex = "^[a-zA-Z\\-' ]+$"
-  val textAreaRegex = """^[a-zA-Z0-9\-'" \t\r\n,.@/]+$"""
-  val textAreaMaxLength = 160
+  protected[forms] val nameRegex = "^[a-zA-Z\\-' ]+$"
```
Comment (kberec) [inline app/forms/TextFormProvider.scala:27]: Do we need `[forms]` here? As it is already directly in forms package?

File: conf/messages.en [hunk 2]
```diff
@@ -626,3 +626,23 @@ companyRecipientName.heading = What is the recipient’s company name?
 companyRecipientName.error.required = Enter the recipient’s company name
 companyRecipientName.error.invalid.characters = Recipient’s company name must only include letters, numbers, and special characters such as hyphens, spaces and apostrophes
 companyRecipientName.error.length = Recipient’s company name must be 160 characters or fewer
+
+isMemberOrConnectedParty.title = Is the recipient a member or a connected party?
```
Comment (kberec) [inline conf/messages.en:630]: Are these under `isMemberOrConnectedParty` used anywhere?

**Review by kberec — APPROVED** [review]

File: app/forms/TextFormProvider.scala (hunk 1 above)
Comment (westwater) [reply app/forms/TextFormProvider.scala:27]: it's mainly so the test in the same package can access the regex to assert and also stops consuming classes from accessing them

File: conf/messages.en (hunk 2 above)
Comment (westwater) [reply conf/messages.en:630]: I think there was an issues when I rebased as I was jumping between the 2 stories (this one and PSR-215) - I'm going to merge PSR-215 after this so I can sort out this issue then

**Pull request merged**